File: src/lib/supabaseClient.js
import { createClient } from '@supabase/supabase-js'

// Credenciales directas (para evitar problemas de variables de entorno)
const supabaseUrl = 'https://rifdbtcnyrwhkqardjxd.supabase.co'
const supabaseAnonKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJpZmRidGNueXJ3aGtxYXJkanhkIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NzMxODQzMzEsImV4cCI6MjA4ODc2MDMzMX0.UlExG1dF-PEi5oW79l1HPOfO5wDnEW6GLKeWQZNDwdc'

// Crear el cliente
export const supabase = createClient(supabaseUrl, supabaseAnonKey)

console.log('✅ Supabase client inicializado')

File: backend/ai_service/supabase_client.py
from supabase import create_client

SUPABASE_URL = "https://rifdbtcnyrwhkqardjxd.supabase.co"

SUPABASE_KEY = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJpZmRidGNueXJ3aGtxYXJkanhkIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3MzE4NDMzMSwiZXhwIjoyMDg4NzYwMzMxfQ.bZkeBAsNy2YQzkkhzIhGStgzqFrahV_u_21H3cDl2BE"


supabase = create_client(
    SUPABASE_URL,
    SUPABASE_KEY
)